ops/playbooks/roles/consdeploy/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+  - name: reload nginx conf file
+    command: "nginx -s reload"
```

Comment:
Not sure if it is working
```
nginx/tasks/main.yml
[3] 1:- name: install nginx
[4] 2:  apt: name=nginx state=latest
[5] 3:- name: start nginx
[6] 4:  service: name=nginx state=started
```
Ansible will start Nginx by triggering `service nginx start` in Ubuntu. I am not sure will this command can really reload a running nginx, I recommend first config test, then do the reload with `service nginx reload`